### PR TITLE
fix: travis errors

### PR DESCRIPTION
**Pull request by quesebifurcan**
Remove hardcoded urls from tests.
Make travis fail early on error.

### CODE REVIEW

File: aether-odk-module/aether/odk/api/tests/test_kernel_utils.py [hunk 1]
```diff
@@ -36,6 +37,8 @@ class KernelUtilsTest(CustomTestCase):
     def setUp(self):
         super(KernelUtilsTest, self).setUp()
 
+        self.aether_kernel_url = os.getenv('AETHER_KERNEL_URL_TEST')
```
Comment (tooxie) [inline aether-odk-module/aether/odk/api/tests/test_kernel_utils.py]: What is the difference between this and ` kernel_url` in line 59?

File: aether-odk-module/aether/odk/api/tests/test_kernel_utils.py [hunk 2]
```diff
@@ -118,7 +119,7 @@ def test__upsert_kernel_artefacts__unexpected_error(self, mock_auth, mock_patch)
         self.assertIn(f'"{str(self.project.project_id)}"', str(kpe.exception), kpe)
         mock_auth.assert_called_once()
         mock_patch.assert_called_once_with(
-            url=f'http://kernel-test:9000/projects/{str(self.project.project_id)}/artefacts/',
+            url=f'{self.kernel_url}/projects/{str(self.project.project_id)}/artefacts/',
```
Comment (obdulia-losantos) [inline aether-odk-module/aether/odk/api/tests/test_kernel_utils.py:122]: :open_mouth:

File: aether-odk-module/aether/odk/api/tests/test_kernel_utils.py [hunk 3]
```diff
@@ -135,7 +136,7 @@ def test__upsert_kernel_artefacts__ok(self, mock_auth, mock_patch):
 
         mock_auth.assert_called_once()
         mock_patch.assert_called_once_with(
-            url=f'http://kernel-test:9000/projects/{str(self.project.project_id)}/artefacts/',
+            url=f'{self.kernel_url}/projects/{str(self.project.project_id)}/artefacts/',
```
Comment (obdulia-losantos) [inline aether-odk-module/aether/odk/api/tests/test_kernel_utils.py:139]: :open_mouth:

Comment (obdulia-losantos) [reply aether-odk-module/aether/odk/api/tests/test_kernel_utils.py:139]: Thanks for catching them! :man_facepalming:

**Review by tooxie — APPROVED** [review]

**Pull request merged**